js/docEnums.js

```diff
@@ -115,6 +115,11 @@
  * @enum {'dash'|'dot'|'longDash'|'solid'}
  */
 
+/**
+ * @typedef {string} Enums.ResizeHandle
+ * @enum {'bottom'|'left'|'right'|'top'|'all'}
```

Comment:
What about "top bottom"/"left right"/ "top bottom"/ "left right bottom" and etc, values?

Reply:
Should not it be declared as `@type Array<Enums.ResizeHandle>` then?

Reply:
> Should not it be declared as @type Array<Enums.ResizeHandle> then?

No, currently several values are [concatenated to a single string](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxResizable/Configuration/#handles).

It doesn't follow from neither new nor previous declaration.
Is this an another task of improving declarations?

Reply:
It requires 14 values to define all applicable combinations. IMHO wrong way. Moreover, on the MVC side, it requires additional code or dictionary to render the spaces between side names.

Reply:
@lukyanovas @IlyaKhD 

In this PR we do not improve cases with options that can accept a combination of accepted values. We just replace `@acceptedValues` to `@enum`. It is enough to solve the issues @jauseg mentioned:
- generate the handles option automatically on DevExtreme MVC Controls side.
- remove manually defined ResizeHandle enum in the SMDG.

I agree with @IlyaKhD that `combination option` support is another task for doc comments improvements and can be done in next PRs.

As for `top bottom|left right|top bottom|left right bottom|etc.`, if we will decide to solve these cases in such manner, we can do this - it will be an extending of the ResizeHandle enum (no BC is expected).